cloudnetwork/v1/register.go

```diff
@@ -7,10 +7,15 @@ import (
 )
 
 var (
-	GroupName          = "cloud.network.openshift.io"
-	SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1"}
-	SchemeBuilder      = runtime.NewSchemeBuilder(addKnownTypes)
-	Install            = SchemeBuilder.AddToScheme
+	GroupName     = "cloud.network.openshift.io"
+	GroupVersion  = schema.GroupVersion{Group: GroupName, Version: "v1"}
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	// Install is a function which adds this version to a scheme
+	Install = SchemeBuilder.AddToScheme
+
+	// SchemeGroupVersion generated code relies on this name
```

Comment:
```suggestion
	// SchemeGroupVersion duplicates GroupVersion and is needed because (some older?) generated code relies on this name
```

as it reads now it can be interpreted as "code generated by SchemeGroupVersion relies on this name"

Reply:
I copy pasted this comment from a bunch of other API groups, they are all consistent now across the board.
I need to double check where this comes from, but I'd suggest we update this universally if we are going to update this

Reply:
yeah i noticed it showed up in a few other spots in your PR, i didn't bother to redundantly comment.

I didn't know it was pulled from elsewhere though. I agree if it's worth fixing it's worth fixing everywhere.  And if it really is in that many places, it may not be worth fixing, so feel free to ignore the suggestion, it just stuck out to me as being a bit confusing as someone who's not super familiar w/ the api codegen process.

Reply:
I'll look into fixing it, but I think I'll do it in a separate PR, it's definitely good feedback